schemas/ontouml2.schema.json

```diff
@@ -88,6 +88,44 @@
         },
         "isDerived": {
           "$ref": "#/definitions/isDerived"
+        },
+        "isExtensional": {
+          "description": "A boolean field that captures if a class decorated as «collective» is extensionally defined (i.e., it's parts cannot change).",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "isPowertype": {
+          "description": "A boolean field that captures if a class decorated as «type» represents a powertype.",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "order": {
+          "description": "A string field that captures the type-order of a class decorated as «type».",
```

Comment:
Does order "0" means the element is an individual? Or this value will be forbidden?

Reply:
Agreed. I'll create an issue for us discuss the representation of objects in the schema.